html_app/ui/MainFrame.js

```diff
@@ -996,118 +991,6 @@ scb.ui.MainFrame = function scb_ui_MainFrame(master_model, context) {
         self.show(state);
     })();
 
-    // init is really not used any more I'll need to move on...
-    self.init = function () {
```

Comment:
Is this strictly required for removing ui_old?  I love deleting code, but if this isn't a direct dependency, it should probably be a different PR so we can revert one without the other if we end up needing it.